src/constants/apimessages.ts

```diff
@@ -0,0 +1,41 @@
+export const appInfoMessages: Record<string, string> = {
+  releaseRetrieved: "Releases Retrieved",
+  contributorDetailsRetrieved: "Contributor Details Retrieved",
+};
+
+export const budgetMessages = {

```

Comment:
Please add object type to all exported objects.

Reply:
done.